apps/yjs/src/yjs-runtime.ts

```diff
@@ -67,6 +67,15 @@ export async function setupRuntimeSocket({
     // FIXME is msg.data a string?
     let { type, payload } = JSON.parse(msg.data as string);
     // console.debug("got message", type, payload);
+    let podId = payload["podId"] || undefined;
+    if (podId) {
+      const oldresult: PodResult = resultMap.get(podId) || { data: [] };
+      if (oldresult.last_exec_end) {
```

Comment:
I think `last_exec_end` is unnecessary, and I intend to remove it. I was not setting it on purpose.

It was only used to clear the results. The result is now (better) cleared on the frontend right after executing the pod ([code](https://github.com/codepod-io/codepod/blob/579ab3b6c37961c89aafc9405f7063608982bda5/apps/ui/src/lib/store/runtimeSlice.ts#L334)).

Reply:
SG, I missed the [code](https://github.com/codepod-io/codepod/blob/579ab3b6c37961c89aafc9405f7063608982bda5/apps/ui/src/lib/store/runtimeSlice.ts#L334) here. Let me clean up the code.